temporalio/exceptions.py

```diff
@@ -70,6 +70,19 @@ def __init__(
         self.run_id = run_id
 
 
+class ApplicationErrorCategory(IntEnum):
+    """Severity category for your application error. Maps to corresponding client-side logging/metrics behaviors"""
+
+    UNSPECIFIED = int(
+        temporalio.api.enums.v1.ApplicationErrorCategory.APPLICATION_ERROR_CATEGORY_UNSPECIFIED
+    )
+
+    """BENIGN category errors emit DEBUG level logs and do not record metrics"""
+    BENIGN = int(
+        temporalio.api.enums.v1.ApplicationErrorCategory.APPLICATION_ERROR_CATEGORY_BENIGN
+    )
+
+
 class ApplicationError(FailureError):
```

Comment:
Can you adjust the default failure converter to properly populate category in both directions (to/from proto)? And can you add (or adjust) a test where, if you raise a benign application error from a workflow, the client can see the category in the application error (it'll be the cause of workflow failed).

Reply:
Yeah - I had done this with go/java and forgot to do so here